README.md

```diff
@@ -1,68 +1,192 @@
+
+**Bobby Hall**
+[<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/)
+
+**Tyler Quinn**
+[<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/)
+
+**Jonathan Picazo**
+[<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/)
+
+**Zachary Peasley**
+[<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/)
+
+**Jarvise Billups-Brown**
+[<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/)
+
+**Emily Arias**
+[<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/)
+
+**Marlon Raskin**
+[<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/)
+
+**Tyler Nishida**
+[<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/)
+
+**Michael Redig**
+[<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/)
+
+## Project Overview
+
+#### Trello Board: 
+https://trello.com/b/Jh8cS46t/conference-contacts
+
+#### Product Canvas: 
+https://www.notion.so/Conference-Contacts-1fa5dafff4964d9d99e07d98dadb8615
+
+#### Design Files:
+https://www.figma.com/file/CkvwWMzxcI4lI4udHm8QiH/Labs18_Conference-Contacts%2C-Emily%2CTylerr?node-id=877%3A72346
+
+# Project Description
+
+A way for people to gather, collect and stay in contact with people that you meet.
+
+To make the social and connecting aspect of events effortless, seamless and fun to do. To give courage and ability to less outgoing people to connect and network more comfortably. To unify the social and personal networking aspect of events and put that in the hands of participants instead of vendors.
+
+## Key Features
+
+- Give users the ability to connect with each other     through the app.
+- Get and Store contacts in a fast and simple way.
+- The ability to message contacts.
+- Find out where you met the contacts.
+
+## Tech Stack
+
+Front end built using:
+
+- React
+- Auth0
+- Reach Router
+- Apollo
+- Graphql
+- Tailwind css
+
```

Comment:
These technologies should be explained below (and only these technologies). Graphql can be taken out since that is what Apollo is doing.